CrisisTextLine/Elmer
# grading schedule observation assignment

#### What's this PR do?
gives credit for 'Schedule Your Observation' assignment upon attending the observation
but, most critically, it rewrote the canvas.js script to use the requestretry library for api requests so we never have to see canvas's 'limit exceeded' messages ever again
#### Where should the reviewer start?
jobs/scheduling/AttendanceSync
canvas.js
#### How should this be manually tested?
global keys and config you know how it is
comment out the cronjob
line 228 change the email, so heather doesn't get spammed
node jobs/scheduling/AttendanceSync.js
also will be useful to run any of the jobs that use canvas library extensively to make sure everything is still working
dropMeltedUsers
NotifyMoreShifts

live laugh love
#### Any background context you want to provide?
#### What are the relevant tickets?
https://admin.crisistextline.org/jira/browse/INT-405
#### Questions:
